packages/mdc-textfield/icon/_mixins.scss

```diff
@@ -55,8 +55,37 @@
   }
 }
 
-@mixin mdc-text-field-icon-color_($color) {
+@mixin mdc-text-field-icon-horizontal-position-two-icons_($position-left, $padding-left, $position-right, $padding-right) {
   .mdc-text-field__icon {
-    @include mdc-theme-prop(color, $color);
+    &:first-of-type {
```

Comment:
Just pointing out this will only work as long as it's safe to assume that the icons always use a distinct type (e.g. `i`, `svg`, etc., not `div`), since `first-of-type` and `last-of-type` apply against the element type, not the selector. I think this is a safe assumption...

Reply:
Maybe add a class like `mdc-text-field__icon--leading` and `mdc-text-field__icon--trailing` can prevent that?

Reply:
I can work around the `last-of-type` issue using a sibling selector
```
.mdc-text-field__icon ~ .mdc-text-field__icon {
    color: red;
}
```